modules/targets/generic-linux.nix

```diff
@@ -48,12 +48,8 @@ in {
     # We need to append system-wide FHS directories due to the default prefix
     # resolving to the Nix store.
     # https://github.com/nix-community/home-manager/pull/2891#issuecomment-1101064521
-    home.sessionVariables = {
-      XCURSOR_PATH = "$XCURSOR_PATH\${XCURSOR_PATH:+:}" + concatStringsSep ":" [
-        "${config.home.profileDirectory}/share/icons"
-        "/usr/share/icons"
-        "/usr/share/pixmaps"
-      ];
+    home.sessionSearchVariables = {
+      XCURSOR_PATH = [ "/usr/share/icons" "/usr/share/pixmaps" ];
```

Comment:
It's configured in `modules/config/home-cursor.nix`.

Reply:
Covered by https://github.com/nix-community/home-manager/pull/4582/files#diff-b07d2165982345df3fc8a099e50b9fb079b473aac705be21d41a765a71ba0546R139-R142.

You can see that there we used the `lib.mkDefault`, that I assume is because we would overwrite this anyway.